services/kaze/routes.go

```diff
@@ -26,7 +26,7 @@ func NewService() http.Handler {
 		AllowAllOrigins:  true,
 		MaxAge:           12 * time.Hour,
 	}
-	router.Use(cors.New(corsConfig), m.FalconGuard())
```

Comment:
As you are merging into the `campus-release` branch isn't `m.FalconGuard()` important ?

Reply:
yeah, It is important.